samples/sensor/qdec/sample.yaml

```diff
@@ -4,10 +4,15 @@ sample:
 common:
   tags: sensors
   timeout: 5
-  harness: console
 
 tests:
   sample.sensor.qdec_sensor:
+    build_only: true
+    platform_allow:
+      - sam_e70_xplained/same70q21
+      - sam_e70_xplained/same70q21b
```

Comment:
Why is this build_only?

Suggest renaming the test to `sample.sensor.sam_qdec_sensor` to follow the convention of other qdec tests.

Reply:
Hi @MaureenHelm 

> Why is this build_only?

I think there is no fixture to run this on HW. I can drop if it is not a problem on CI.

> Suggest renaming the test to `sample.sensor.sam_qdec_sensor` to follow the convention of other qdec tests.

I kept the name generic because I assume that any hardware that have this functionality and don't have fixture can be checked at CI using this generic entry. Please, signal to me if I should keep generic or change to your suggestion : )

Reply:
I relaxed the list to any that have the alias QDEC0 and ran locally without errors. Let's see the output from CI .
It was necessary keep a special entry for SAM because boards got skipped since they are not `integration_platforms` I presume.

Reply:
Hi @MaureenHelm ,

CI is green, I wondering if still anything missing.